### PR TITLE
Quite Flag Added

### PR DESCRIPTION
**Pull request by serkanerip**
Too much output printing stdout and i think this tool should have a quite flag to just print graph output.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ func init() {
 	log.SetOutput(os.Stderr)
 	// add flags here
 	//rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	rootCmd.Flags().BoolP("quite", "q", false, "Only display graph output.")
```
Comment (ahmetb) [inline main.go:50]: you mean -quiet :)

**Comment by ahmetb** [comment]
I wouldn't bother with this for now. But thanks for the PR.

I think I'll be doing stuff like `-svg-out` `-dot-out` in the future. And the default will be `-quiet` anyway and it'll just launch a browser. 👍🏼

**Comment by serkanerip** [comment]
No problem i get fun while doing :)

File: main.go (hunk 1 above)
Comment (serkanerip) [reply main.go:50]: ups, typo :)

**Pull request closed without merging**